README.rst

```diff
@@ -1225,8 +1225,12 @@ Then, add the following command to your CI:
 
 .. code-block:: bash
 
-    $ IFS=$'\n'; COMMIT_SCA_FILES=($(git diff --name-only --diff-filter=ACMRTUXB "${COMMIT_RANGE}")); unset IFS
-    $ vendor/bin/php-cs-fixer fix --config=.php_cs.dist -v --dry-run --stop-on-violation --using-cache=no --path-mode=intersection -- "${COMMIT_SCA_FILES[@]}"
+    $ CHANGED_FILES=`git diff --name-only --diff-filter=ACMRTUXB $COMMIT_RANGE`
+    $ if ! echo "$CHANGED_FILES" | grep -qE "^(\.php_cs\.dist|composer\.lock)$"
+    $ then
+    $    IFS=$'\n' FIX_ARGS=('--path-mode=intersection' '--' ${CHANGED_FILES[@]})
```

Comment:
Nope: when a variable definition is prepended to a command, it only applies to this command.